starter_templates/csharp/src/Server.cs

```diff
@@ -6,5 +6,8 @@
 
 // Uncomment this block to pass the first stage
 // TcpListener server = new TcpListener(IPAddress.Any, 6379);
+// // Since the tester restarts your program quite often, setting SO_REUSEADDR
+// // ensures that we don't run into 'Address already in use' errors
+// server.Server.SetSocketOption(SocketOptionLevel.Socket, SocketOptionName.ReuseAddress, true);
```

Comment:
@ryan-gang interesting, do we know why this wasn't a problem before? I'm looking at the code examples for C# in stage 2 and I don't see any usages like this (checked 3).

Reply:
Removed the C# update, had some doubts myself also. 
That seems to be the safer option.